README.md

```diff
@@ -97,6 +97,12 @@ foo,bar
 10,20
 ```
 
+```shell
+❯ pqrs cat data/simple.parquet --csv-no-header
```

Comment:
thanks a lot for the contribution @jeffbski-rga! I think it would be better to have `--no-header` as a separate flag instead?

something like:

`--csv`: current csv format, with header
`--csv --no-header`: csv format without headers

thoughts?

Reply:
Sure, I should be able to make the change to use separate flag.

Reply:
thanks!

Reply:
ok, see how this looks.